README.md

```diff
@@ -157,7 +183,15 @@
 ## Dreamweaver
 
 - ✨ [Netbeans](https://netbeans.apache.org/) *(Windows, Mac, Linux)*
+- ✨ [Eclipse](https://eclipse.org/) *(Windows, Mac, Linux)*
+- ✨ [Aptana Studio](http://www.aptana.com/) *(Windows, Mac, Linux)*
 - ✨ [VSCodium](https://vscodium.com/) *(Windows, Mac, Linux)*
+- ✨ [Pulsar](https://pulsar-edit.dev/) *(Windows, Mac, Linux)*
+- ✨ [Vim](https://www.vim.org/download.php) *(Windows, Mac, Linux)*
+- ✨ [NeoVim](https://neovim.io/) *(Windows, Mac, Linux)*
+- ✨ [Emacs](https://www.gnu.org/software/emacs/) *(Windows, Mac, Linux)*
+- ✨ [Notepad++](https://notepad-plus-plus.org/) *(Windows)*
```

Comment:
```suggestion
```

I think this ones are more text editors than a good alternative to Dreamweaver

Reply:
Notepad++ has all features a code editor need to do websites. It's not just a text editor.

Reply:
> I think this ones are more text editors than a good alternative to Dreamweaver

That's true, but statistically speaking most developers are using text editors for web development. Dreamweaver is much more of an IDE then it is a text editor. In my experience most web developers have switched to using something that is more of a text editor than an IDE like Visual Studio or something. Both are valid ways of doing web development, so maybe I could add a section for IDEs and a section for text editors?

Reply:
I'm not sure it's worth complicating the list even more, maybe just adding a comment next to each software to indicate if it's an IDE or a text editor, but I think interested people will see it quickly by clicking on the links.

Reply:
> I'm not sure it's worth complicating the list even more, maybe just adding a comment next to each software to indicate if it's an IDE or a text editor, but I think interested people will see it quickly by clicking on the links.

I'm fine with either, just lemme know what you all want and I'll update the PR. 😄